experimental/dds/attributable-map/src/mapKernel.ts

```diff
@@ -386,29 +386,23 @@ export class AttributableMapKernel {
 	}
 
 	private setAttribution(key: string, message?: ISequencedDocumentMessage): void {
```

Comment:
This method could use some documentation.  Under what circumstances is `message` passed, and what's the significance?  Probably also inline-comment the conditionals below to explain the differences in what gets set into the attribution, and what they signify.